thentos-adhocracy/src/Thentos/Adhocracy3/Types.hs

```diff
@@ -0,0 +1,103 @@
+{-# LANGUAGE DataKinds                   #-}
+{-# LANGUAGE DeriveDataTypeable          #-}
+{-# LANGUAGE FlexibleContexts            #-}
+{-# LANGUAGE FlexibleInstances           #-}
+{-# LANGUAGE MultiParamTypeClasses       #-}
+{-# LANGUAGE OverloadedStrings           #-}
+{-# LANGUAGE ScopedTypeVariables         #-}
+{-# LANGUAGE TypeFamilies                #-}
+{-# LANGUAGE TypeOperators               #-}
+{-# LANGUAGE UndecidableInstances        #-}
+
+
+module Thentos.Adhocracy3.Types
+  ( module Thentos.Types
+  , module Thentos.Adhocracy3.Types
+  )
+  where
+
+import Control.Monad.Except (MonadError, throwError)
+import Data.Data (Typeable)
+import Data.String.Conversions (LBS)
+
+import Thentos.Types
+
+
+-- FIXME: workaround implementation until #193 is fixed:
+
+data ThentosError' =
+      A3BackendErrorResponse Int LBS
+    | A3BackendInvalidJson String
+  deriving (Eq, Show, Typeable)
+
+throwA3Error :: ThentosError' -> a
+throwA3Error = error . show
```

Comment:
correction: the type *should* be

```haskell
throwA3Error :: (e ~ ThentosError', MonadError e m) => e -> m a
```

but that does not type.  it will type once #193 has been fixed, so i'd like to leave this as is for now.